src/api/api.js

```diff
@@ -0,0 +1,23 @@
+import { push } from "../routes/router.js";
+
+const API_END_POINT = "https://kdt-frontend.programmers.co.kr";
+
+export const request = async (url, options = {}) => {
+  try {
+    const res = await fetch(`${API_END_POINT}${url}`, {
+      ...options,
+      headers: {
+        "Content-Type": "application/json",
+        "x-username": "colorkite10",
+      },
```

Comment:
P1;

이렇게 하게 된다면 headers를 변경하고 싶어도 변경하지 못할 것 같아요.
아래와 같이 headers 또한 수정 가능하게 해두면 어떨까요? 

```suggestion
      ...options,
      headers: {
        "Content-Type": "application/json",
        "x-username": "colorkite10",
        ...options?.headers
      },
```